### ISSUE TITLE
Focusing on building mutual understanding

### ISSUE THREAD

**Issue by chrisgraham** [ContributorCovenant/contributor_covenant#795]
Commonly people conflict because they don't take the time to understand each other's perspectives.

I'd consider adding another positive behavior to the list to encourage this:

> Rather than assuming a lack of insight in people, communicate to understand why people have different perspectives than yours and try and reach shared broader understandings

**Comment by chrisgraham** [comment]
I talked to a colleague and they independently had a concern about egoistical and impatient answers from people in OSS communities, and we agreed it could be covered by this issue if we expand the wording a little...

> Rather than assuming a lack of insight in people, give people the benefit of the doubt, communicate to understand why people have different perspectives than yours, and try and reach shared broader understandings

**Comment by max-keviv** [comment]
@chrisgraham  can i take this

**Comment by chrisgraham** [comment]
If you mean you want to use my wording somewhere, absolutely. I don't claim any copyright or require any credit for anything I've posted here. If you mean you want to somehow contribute changes to the project, that's something I have no control over personally.

**Comment by rudiservo** [comment]
> I talked to a colleague and they independently had a concern about egoistical and impatient answers from people in OSS communities, and we agreed it could be covered by this issue if we expand the wording a little...
> 
> > Rather than assuming a lack of insight in people, give people the benefit of the doubt, communicate to understand why people have different perspectives than yours, and try and reach shared broader understandings

good idea but I think it wont pass, it is not enforcing enough to people to accept they can't disagree, although I would love to be mistaken!
